### ISSUE TITLE
ERROR should be INFO?

### ISSUE THREAD

**Issue by SlimeDog** [dniym/IllegalStack#129]
**Required Information**
```
>version IllegalStack
[22:35:25] [Server thread/INFO]: IllegalStack version 2.4.0
[22:35:25] [Server thread/INFO]: Closes some dupe bugs still present in vanilla  minecraft.
[22:35:25] [Server thread/INFO]: Website: https://www.spigotmc.org/resources/44411/
[22:35:25] [Server thread/INFO]: Author: dNiym
>version
[22:35:46] [Server thread/INFO]: This server is running CraftBukkit version 3181-Spigot-38e6c03-ed2b91c (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT)
[22:35:46] [Server thread/INFO]: You are running the latest version
```

**Describe the bug**
IllegalStack reports an ERROR for a configured exception. Given that I have configured
```
Exploits:
  OverStack:
    AllowStack:
    - POTION
```
this should not be an ERROR. Perhaps an INFO message.
```
[21:58:03] [Server thread/INFO]: [IllegalStack] Enabling IllegalStack v2.4.0
[21:58:03] [Server thread/ERROR]: The following materials are allowed to have stacks larger than the vanilla size: POTION
```

**To Reproduce**
1. Install IllegalStack 2.4.0-b52
2. Configure a stacking exception, per above
3. Start the server

**Checklist**:

- [x] I included all information required in the sections above
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/dniym/IllegalStack/issues?q=is%3Aissue+is%3Aopen+)
- [x] I made sure I am using an up-to-date version of [IllegalStack](https://www.spigotmc.org/resources/44411/)
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by dniym** [comment]
Ah yes an oversight I'll get that fixed ASAP

**Comment by dniym** [comment]
Fixed as of version 2.4.1

**Comment by SlimeDog** [comment]
Confirmed fixed in 2.4.1. Thanks.
